models/task/task.go

```diff
@@ -31,9 +31,7 @@ func NewTask(id int64, listID int64, userID int64, issueNumber sql.NullInt64, ti
 	return task
 }
 
-// TODO: ここerrorを返すようにしなくていいのか要検証
-// いやエラー返せよ
-func FindTask(listID int64, taskID int64) *TaskStruct {
+func FindTask(listID int64, taskID int64) (*TaskStruct, error) {
```

Comment:
__[golint]__
* [golint] exported function FindTask should have comment or be unexported